src/Category/Applicative.agda

```diff
@@ -12,29 +12,29 @@
 module Category.Applicative where
 
 open import Level using (Level; suc; _⊔_)
-open import Data.Unit
+open import Data.Unit.Polymorphic
 open import Category.Applicative.Indexed
 
 private
   variable
     f : Level
 
 RawApplicative : (Set f → Set f) → Set (suc f)
-RawApplicative F = RawIApplicative {I = ⊤} λ _ _ → F
+RawApplicative {i} F = RawIApplicative {I = ⊤ {i}} λ _ _ → F
 
 module RawApplicative {F : Set f → Set f}
                       (app : RawApplicative F) where
   open RawIApplicative app public
 
 RawApplicativeZero : (Set f → Set f) → Set _
-RawApplicativeZero F = RawIApplicativeZero {I = ⊤} (λ _ _ → F)
+RawApplicativeZero {i} F = RawIApplicativeZero {I = ⊤ {i}} (λ _ _ → F)
```

Comment:
Actually, any chance you could explain the point of this change? I'm probably being slow but why does it matter what level the indexing type is at? Given that the indexing is trivial anyway...

Reply:
You're quite right, it's over-generalized.